src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1266,3 +1266,29 @@ ingress-nginx:
 ```
 
 The full list of values is [available here](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml).
+
+#### Okteto ingress-nginx default values
+
+Okteto Self Hosted applies some defaults to ingress-nginx, which can be checked before overriding them with the following command:
```

Comment:
can we be specific on the defaults that are applied?  Is this for all ingress-nginx, or just the one installed by us?